_layouts/blog.html

```diff
@@ -1,16 +1,22 @@
 ---
-layout: default
+layout: page
```

Comment:
Okay just so we are on the same page, do you want the header image to show up on the other pages? Or do want each page to have it's own header image? Or ultimately is your goal to make the website a single scrolling page website?